src/app/globals.css

```
@@ -184,4 +184,8 @@ h1, h2, h3, h4, h5, h6 {
   .hero-bg {
     background-image: url('/assets/HeroSectionTempleDesk3.png?v=1') !important;
   }
+
+  .mr {
+    
+  }
```

Stylelint (17.2.0)

[error] Unexpected empty block (block-no-empty)

(block-no-empty)